changelog.adoc

```diff
@@ -1,5 +1,8 @@
 = Changelog
 
+== 2.0.0
+* add capability to update from log4j 1.2 to log4j 2
```

Comment:
at the moment, I would add the marker : WINDOWS ONLY, should we?

Reply:
The tool works on Linux as well. We've already used it.